docs/development.md

```diff
@@ -128,8 +128,9 @@ You need a student account which can be created by instructors.
 
 ## Testing
 
-TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
-It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).
+* TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
+  * It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).
```

Comment:
Let's remove this line as this is no longer the case, although the Chrome version is a little less unstable.

Reply:
You mean the Firefox version? I think it is still good to mention at least a range of version, I used Firefox 57.0.2 and I don't think the selenium worked.

Reply:
The thing is we are no longer only running tests during CI with Firefox, Travis uses Firefox 46.0 and AppVeyor uses ChromeDriver.

In the ideal case, tests should pass for both but right now it is not the case.

So if we leave the original line here, we are not providing correct information.

However, if we decide to modify this to state the current state of our tests (currently flaky), we have to continue to update the docs until we reach the ideal case.

And for the ideal case, the tests should work for Firefox (up to `46.0.1`) and Chrome which is already indicated by the documentation below.

Therefore the two better course of actions is to either modify or delete, which I think the better option among the two is to delete and let the builds show for themselves (continually maintaining the docs to show progress is probably not a good idea).